nn/transformer.py

```diff
@@ -69,15 +69,17 @@ class TransformerEncoder(nn.Module):
   Defines the full Encoder of the standard transformer
   """
   def __init__(self, encoder_layer: Union[TransformerEncoderLayer, Any], num_layers: int,
-               norm=nn.layer_norm, norm_eps: float = 1e-5):
+               norm=nn.layer_norm, norm_eps: float = 1e-5, pos_enc: Callable = nn.positional_encoding):
```

Comment:
`Callable` is the wrong type.

Reply:
What instead would you suggest? I have not really dealt with positional encodings in my work yet, so I dont really have good knowledge on what would be better. I tried to have a look at other Frameworks and tried to replicate it here.